### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by DSsoto**
Just improving the juju of our codebase.

### CODE REVIEW

File: perception/mil_vision/CMakeLists.txt [hunk 1]
```diff
@@ -1,34 +1,34 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(mil_vision)
 
-# set c++11 as default, overide with set_target_properties()
+# set c++11 as default, overide with target_vompile_options()
```
Comment (kev-the-dev) [inline perception/mil_vision/CMakeLists.txt]: Gatta fix those commented out bugs

File: perception/mil_vision/exFAST_SparseStereo/src/example/example.cpp [hunk 2]
```diff
@@ -1,144 +0,0 @@
-/*
```
Comment (kev-the-dev) [inline perception/mil_vision/exFAST_SparseStereo/src/example/example.cpp:1]: Why are we removing this? No chance of every using it?

**Review by kev-the-dev — CHANGES_REQUESTED** [review]
Will approve when I see it in CI. Note, should be merged simultaneously with uf-mil/SubjuGator#218 . Good stuff

File: perception/mil_vision/exFAST_SparseStereo/src/example/example.cpp (hunk 2 above)
Comment (DSsoto) [reply perception/mil_vision/exFAST_SparseStereo/src/example/example.cpp:1]: I didn't like its performance, will be replacing with a better alternative.

File: perception/mil_vision/CMakeLists.txt (hunk 1 above)
Comment (DSsoto) [reply perception/mil_vision/CMakeLists.txt]: Fixed

File: drivers/mil_blueview_driver/CMakeLists.txt [hunk 3]
```diff
@@ -1,6 +1,6 @@
 cmake_minimum_required(VERSION 2.8.3)
 SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
-project(blueview_driver)
+project(mil_blueview_driver)
```
Comment (sentree) [inline drivers/mil_blueview_driver/CMakeLists.txt:3]: @ironmig broke naming convention - sad.

File: utils/mil_tools/CMakeLists.txt [hunk 4]
```diff
@@ -8,20 +8,19 @@ find_package(catkin REQUIRED COMPONENTS
   rospy
   roscpp
   cv_bridge
-  cmake_modules
 )
 
-find_package(Eigen 3 REQUIRED)
+find_package(Eigen3 REQUIRED)
```
Comment (sentree) [inline utils/mil_tools/CMakeLists.txt:13]: Fixing this everywhere is going to look incredible on builds, thanks!

File: utils/mil_tools/src/mil_tools/mil_tools.cpp [hunk 5]
```diff
@@ -1,6 +1,6 @@
 #include <mil_tools/mil_tools.hpp>
-#include <mil_tools/msg_helpers.h>
-#include <mil_tools/param_helpers.h>
+#include <mil_tools/msg_helpers.hpp>
+#include <mil_tools/param_helpers.hpp>
```
Comment (sentree) [inline utils/mil_tools/src/mil_tools/mil_tools.cpp:3]: Have you tried using `find` to look for these? You could be sure to get them all.

**Comment by sentree** [comment]
I remember you saying that this and uf-mil/SubjuGator#218 were codependent; does this require that pull request to build? We want to keep mil_common pure, so this should not depend on that at all.

**Review by sentree — APPROVED** [review]
Really liking the level of OCD here. As @ironmig said, this is good to merge once it builds.

**Comment by DSsoto** [comment]
@whispercoros the changes are co-dependent, mil_common is not dependent on Sub

Were you able to build them together on a clean Docker image?

**Comment by sentree** [comment]
This passed build, but only ran the BlueView unit tests. I'm not sure why but it looks like the ROS alarms tests didn't even build for this and #32. Check the logs and tell me if you see anything fishy.

**UPDATE:** I figured this out - the `checkout scm` command doesn't pull submodules. I will add that step to the pipeline later tonight.

**Comment by sentree** [comment]
A set of codependent repositories would mean that both depend on each other (i.e. mil_common and odometry_tools). This is just SubjuGator depending on mil_common, which is normal.

**Review by kev-the-dev — APPROVED** [review]

**Pull request merged**